README.md

```diff
@@ -44,12 +44,16 @@ or
 **Documentation**: [current release](https://3yourmind.github.io/kotti/) or [current master](https://kotti.netlify.com/)
 
 ```typescript
-// in main.js / entrypoint
+// in main.ts / entrypoint
 import Vue from 'vue'
 import KottiUI from '@3yourmind/kotti-ui'
 import '@3yourmind/kotti-ui/dist/styles.css'
 
-// register all KtComponents globally
+// also make sure to set-up @vue/composition-api as it’s required for newer Kotti features
+import VueCompositionAPI from '@vue/composition-api' // right now, 0.5.0 is recommended
```

Comment:
we specify it to be 0.5.x on our peer deps
maybe we shouldn't be that specific and the best we can add is 'pls no use of beta'

Reply:
so even higher patches are supported and i think we already have that in yoda if i'm not mistaken

just saying maybe let's not overkill with specificity but also leave room for a beta-fuckup

Reply:
not an essential change just a suggestion and the last i have 

so i will approve already

Reply:
https://github.com/3YOURMIND/kotti/blob/cc115881976692d3d14504ab2430fb1a3a63f8ab/packages/kotti-ui/package.json#L50